flexx/ui/_widget.py

```diff
@@ -1158,7 +1158,9 @@ def _comp_init_property_values(self, property_values):
         # when this is the active component, and after the original
         # version of this has been called, everything related to session
         # etc. will work fine.
-
+        # Property values must be poped when consumed so that the remainer is used for 
+        # instantiation of the Widget
```

Comment:
Could you move this comment down to where `super()._comp_init_property_values(property_values)` is called to say that this call will pop the values ?

Reply:
I've added the comment all along the trail... It should be clear now. :)